scripts/variantstore/wdl/ngs_filter_extract.wdl

```diff
@@ -154,7 +160,8 @@ workflow NgsFilterExtract {
         indel_recal_tranches = IndelsVariantRecalibrator.tranches,
 
         fq_info_destination_table = fq_filter_set_info_table,
-        fq_tranches_destination_table = fq_filter_set_tranches_table
+        fq_tranches_destination_table = fq_filter_set_tranches_table,
```

Comment:
i know this isn't a change in your PR, but while we're in here, could we change `fq_filter_set_tranches_table` to `fq_tranches_destination_table` everywhere? that would clarify things so much for me (and +1 for consistency)